Extensions/accesskit.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE AccessKit **//
-//* VERSION 1.1 REV D **//
+//* VERSION 1.1.5 **//
```

Comment:
I took "D" to be equal to ".4", and bumped it up from there.

On Wed, Aug 12, 2015, 01:56 Bryan Tsang <notifications@github.com> wrote:

> In Extensions/accesskit.js
> <https://github.com/new-xkit/XKit/pull/220#discussion_r36814103>:
>
> > @@ -1,5 +1,5 @@
> >  //* TITLE AccessKit **//
> > -//* VERSION 1.1 REV D **//
> > +//* VERSION 1.1.5 **//
>
> Is there any reason for the large bump in patch version? This might be a
> nitpick, but I'd prefer either 1.1.0 or 1.1.1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/220/files#r36814103>.
>